routing/index_graph_starter.hpp

```diff
@@ -91,6 +110,14 @@ class IndexGraphStarter final
   void FindPointsWithCommonFeature(Joint::Id jointId0, Joint::Id jointId1, RoadPoint & result0,
                                    RoadPoint & result1);
 
+  void AddAdditionalZeroEdges(FakeJoint const & fp, EndPointType endPointType);
+
+  // Edges which added to IndexGraph in IndexGraphStarter. Size of |m_additionalZeroEdges| should be
+  // relevantly small because some methods working with it have linear complexity.
+  vector<DirectedEdge> m_additionalZeroEdges;
+  // |m_jointsOfAdditionalEdges| is set of all joint ids take part in |m_additionalZeroEdges|.
+  set<Joint::Id> m_jointsOfAdditionalEdges;
+  uint32_t m_additionalNextFeatureId;
```

Comment:
We can have only "real" features (edges) and "fake" features (edges). That's it. But "fake" feature of IndexGraphStarter and IndexGraph are not the same. And it's quite logical. If we need to add a feature to graph, we call it a fake feature.